safe/utilities/keyword_io.py

```diff
@@ -670,3 +678,83 @@ def get_statistics(self, layer):
             statistics_classes = {}
 
         return statistics_type, statistics_classes
+
+
+def write_to_new_keyword(source, keywords):
```

Comment:
Can you change these method names to:

write_iso_keywords
read_iso_keywords ?